src/components/Login/Login.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Header from "./Header/Header";
+import MainContent from "./MainContent/MainContent";
+
+const Login = ({ onThemeToggle, mode, theme }) => {
+  return (
+    <>
+      <Header onThemeToggle={onThemeToggle} mode={mode} />
+      <MainContent mode={mode} theme={theme} />
```

Comment:
`MainContent` 🤔, I thought it will load main content of home..... Probably you can name it login card etc

Reply:
changed component and file names as per suggestion